sherpa/optmethods/optfcts.py

```diff
@@ -1153,7 +1153,7 @@ class fdJac:
         def __init__(self, func, fvec, pars):
             self.func = func
             self.fvec = fvec
-            epsmch = numpy.float_(numpy.finfo(numpy.float).eps)
+            epsmch = numpy.float_(numpy.finfo(float).eps)
```

Comment:
Again, I don't see what this is explicitly typecast at all.
```suggestion
            epsmch = numpy.finfo(float).eps
```